data/processed/hesa/gbp_research_income.yaml

```diff
@@ -0,0 +1,24 @@
+api: fetch
+api_documentation: https://www.hesa.ac.uk/data-and-analysis/estates/table-1
+description: Research income received by universities in the NUTS2 region
+endpoint_static: https://www.hesa.ac.uk/data-and-analysis/estates/data.csv
+framework_group: public_rnd_capability
+is_experimental: False
+nuts_level: 2
+source: HESA (Higher Education Statistical Agency) https://www.hesa.ac.uk/
+year_range: [2015,2017]
+
```

Comment:
extra line to be removed, also (minor) it would be great placing these `order:` lines in alphabetical order (after `nuts_level`)